patches/rrweb@2.0.0-alpha.12.patch

```diff
@@ -0,0 +1,522 @@
+diff --git a/es/rrweb/packages/rrweb/src/replay/index.js b/es/rrweb/packages/rrweb/src/replay/index.js
+index e9a8ab2ba94093198f3dc42c9f6c4915f99cbc1d..182662fff8cca2eb4c63d956f0fce94604ce1aa1 100644
+--- a/es/rrweb/packages/rrweb/src/replay/index.js
++++ b/es/rrweb/packages/rrweb/src/replay/index.js
+@@ -201,6 +201,10 @@ class Replayer {
+             mouseTail: defaultMouseTailConfig,
+             useVirtualDom: true,
+             logger: console,
++            onError: (e) => {
++                // maintain the existing behaviour of throwing if no handler is provided
++                throw e;
++            },
```

Comment:
Very difficult to parse this patch but it's just this change and adding a try / catch